WordPress/src/main/res/values/reader_styles.xml

```diff
@@ -151,6 +157,18 @@
         <item name="android:textColor">?attr/colorOnSurface</item>
     </style>
 
+    <style name="ReaderTextView.Post.Welcome.Title" parent="ReaderTextView">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:textAppearance">?attr/textAppearanceHeadline6</item>
+        <item name="android:textAlignment">center</item>
+        <item name="android:textColor">@color/reader_welcome_card_text</item>
+        <item name="android:fontFamily">serif</item>
+        <item name="android:text">@string/reader_welcome_banner</item>
```

Comment:
I'm wondering if `text` can be considered as a style. I'd personally move it to the layout file or perhaps even the VM. Wdyt